Testing/SystemTests/tests/framework/SNSPowderRedux.py

```diff
@@ -139,7 +142,14 @@ def runTest(self):
                            ContainerShape="PAC06",
                            OutputFilePrefix='PP_absorption_',
                            OutputDirectory=savedir)
+
+        # Check name, number density, effective density
         assert mtd['PG3_46577'].sample().getMaterial().name() == 'Si'
+        assert mtd['PG3_46577'].sample().getMaterial().numberDensity == 0.049960265513165146
```

Comment:
These material constants should be pulled into a file constant since they are used in a couple of places.